src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -143,6 +143,10 @@ constexpr Register x19_sender_sp = x19; // Sender's SP while in interpreter
 constexpr Register t0 = x5;
 constexpr Register t1 = x6;
 constexpr Register t2 = x7;
+constexpr Register t3 = x28;
+constexpr Register t4 = x29;
+constexpr Register t5 = x30;
+constexpr Register t6 = x31;
 
```

Comment:
In your case it doesn't look like we need them?

So I think you should revert these changes.
As we may want to reserve one of those registers for something in the future.
I don't think we should take lightly on just start using them.

Reply:
Not sure I understand this argument. We can still use the registers using `x[28-31]`. Why not give them their more informative name? Also, I do use them in the MD5 intrinsic, FWIW.

Reply:
Yes, I see what you mean.
I think I was just wrong.